plugins/redmine_bugs_ruby_lang/lib/redmine_mailing_list_integration/redmine_ext/mailer.rb

```diff
@@ -1,27 +1,21 @@
 module MailingListIntegrationMailer
 
-  def issue_add(user, issue)
+  def issue_add_mailing_list(user, issue)
     mailing_lists = issue.project.mail_routes_for_issue(issue)
-
     record_message(issue, nil, mailing_lists)
-
-    m = super(user, issue)
+    m = issue_add(user, issue)
 
     m.header[:to] = mailing_lists.map(&:address)
-    m.header[:subject] = "[#{issue.project.name} #{issue.tracker.name}##{issue.id}] #{issue.subject}"
```

Comment:
if you want a custom subject just for mailing lists we should keep it here. I moved it to  `RubyLangMailingListCustomizationMailer` which is more appropriate I think.